hw/ip/tlul/data/xbar_tests.hjson

```diff
@@ -0,0 +1,169 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  run_modes: [
+    {
+      name: xbar_mode
```

Comment:
I think this requires an explanation - the IP level XBAR testbenches do not use this, only the chip level one does.

Reply:
Sure, added commment.